app/services/NotificationLists.scala

```diff
@@ -41,6 +50,42 @@ object NotificationLists {
     case _ => Either.right(atom)
   }
 
+  def sendNotificationList(atom: Atom): Either[AtomAPIError, Atom] = atom.data match {
+    case AtomData.Storyquestions(data) => {
+      val result = for {
+        userQuestions <- data.editorialQuestions
+        questionSet <- userQuestions.headOption //We only use the first questionSet in the atom
+      } yield {
+        val results = for {
+          question <- questionSet.questions
+          if question.answers.nonEmpty
+        } yield {
+          val qa = QuestionAnswers.fromThrift(atom.id, question)
+          val request = new InvokeRequest()
+            .withFunctionName(Config.lambdaFunctionName)
+            .withPayload(qa.asJson.toString)
+            .withLogType(LogType.Tail)
+          lambdaClient.invoke(request)
+        }
+  
+        if (results.forall(_.getStatusCode == 200))
+          Either.right(atom)
+        else {
+          results
+            .foreach { r => 
+              Logger.error(s"Unable to trigger email notifications (${r.getStatusCode})")
+              Logger.error(new String(Base64.getDecoder.decode(r.getLogResult), StandardCharsets.UTF_8))
```

Comment:
Is it an error (in this application) to get a failed response from an API?

I don't know if you have alarms that trigger every time an error is logged, but I wouldn't want to be called because an API is down for a few seconds...

Reply:
Good point. I'll take this as a UX improvement since it will be propagated to the UI.